frontend/src/app/core/services/notification.service.ts

```diff
@@ -11,14 +12,22 @@ export class NotificationService {
 
     private notificationModal: NgbModalRef;
 
+    private notificationSubject = new BehaviorSubject(this.notifications);
+
+    currentNotifications = this.notificationSubject.asObservable();
```

Comment:
private after public

Reply:
I refactored some properties with public modifier because 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/93576084/3866571a-f0f1-4aac-a77b-027cfc8a1059)